bin/updatestrings.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+../../../app/console translation:extract en -v \
```

Comment:
maybe change to run from project root as they seems to depend on that anyway? _(and check if in root while at it, this is placed in public folder, aka no `.<folder>` like `.travis`, so should probably do some sanity checks before running)_